src/Shapefile.jl

```diff
@@ -4,49 +4,124 @@ import GeoInterface, DBFTables, Tables
 
 using RecipesBase
 
+# Types: could we move these to a shapes.jl file?
+
+"""
+    Rect
+
+A rectangle object to represent the bounding box for other shape file shapes.
+"""
 struct Rect
     left::Float64
     bottom::Float64
     right::Float64
     top::Float64
 end
 
+function Base.:(==)(a::Rect, b::Rect)
```

Comment:
Was this also part of the unrelated PR? Maybe also a TODO to couple this with Extents.jl?